airbyte-cdk/python/airbyte_cdk/sources/declarative/decoders/noop_decoder.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+import logging
+from typing import Any, Generator, Mapping
+
+import requests
+from airbyte_cdk.sources.declarative.decoders.decoder import Decoder
+
+logger = logging.getLogger("airbyte")
+
+
+class NoopDecoder(Decoder):
+    def is_stream_response(self) -> bool:
+        return False
```

Comment:
should be True to pass `request_kwargs = {"stream": True}` into download request

Reply:
This decoder is only used for the paginator 
[here](https://github.com/airbytehq/airbyte/pull/45678/files#diff-0572edeedd561901f8110c0f88b48efde87d1ed91247c0eb189c7229da12090dR648) and [here](https://github.com/airbytehq/airbyte/pull/45678/files#diff-0572edeedd561901f8110c0f88b48efde87d1ed91247c0eb189c7229da12090dR658). We won't use this decoder for the download retriever